Sources/SwiftInspectorCore/FileManager+SwiftFiles.swift

```diff
@@ -55,4 +55,12 @@ extension FileManager {
 
     return swiftFiles
   }
+
+  /// Checks whether a URL contains a Swift file
+  ///
+  /// - Parameter baseURL: The URL to look for a Swift file
+  /// - Returns: `true` if the URL contains a Swift file, `false` otherwise
+  public func isSwiftFile(at baseURL: URL) -> Bool {
+    FileManager.default.fileExists(atPath: baseURL.path) && !baseURL.hasDirectoryPath && baseURL.pathExtension == "swift"
```

Comment:
FWIW I suspect that actually checking if it's a directory with the file manager is more robust.
I bet that if you have a directory named `/files/MyDirectory.swift` this would succeed erroneously.
Could be worth adding a test, but depends on how paranoid we want to be.

Reply:
Good call! it actually works! See https://github.com/fdiaz/SwiftInspector/pull/19/commits/2773c52ee6d508bf450a07b7ba6e38ccf942a13f

Reply:
Wow crazy! I wonder how it knows.